src/topology/homotopy/homotopy_group.lean

```diff
@@ -168,18 +176,19 @@ begin
     to_continuous_map_comp_symm, continuous_map.comp_id], ext, refl,
 end
 
-/-- The (n+1)-dimensional loops are isomorphic to the loop space at `const`.-/
+/-- The `n+1`-dimensional loops are equivalent to the loop space at `const` as a `n`-dimensional
+  loop, defined abtractically for any nonempty index type `N`. -/
```

Comment:
```suggestion
  loop. We allow an arbitrary indexing type `N` in place of `fin n` here. -/
```
(We may also put the last sentence in the module docstring so we don't need to repeat it in every declaration.)